### PR TITLE
in recreate_theme_previews run generate_static_theme_preview with adhoc queue

### LINKED ISSUE #16973: run the theme preview regeneration tasks in a different task queue
follow up for #16920

we generate theme previews in the priority queue because normally we want to display them to the theme developer straight away. But if we do this for `recreate_theme_previews` the priority queue ends up full of themes, which blocks other priority tasks from completing (including elasticsearch reindex tasks leading to a stale index).

### PR DESCRIPTION
**Pull request by eviljeff**
fixes #16973

### CODE REVIEW

File: src/olympia/addons/tasks.py [hunk 1]
```diff
@@ -243,7 +240,9 @@ def recreate_theme_previews(addon_ids, **kw):
             log.info('Recreating previews for theme: %s' % version.addon_id)
             xpi = get_filepath(version.all_files[0])
             theme_data = parse_addon(xpi, minimal=True).get('theme', {})
-            generate_static_theme_preview(theme_data, version.id)
+            generate_static_theme_preview.apply_async(
+                args=(theme_data, version.id), queue='adhoc'
```
Comment (eviljeff) [inline src/olympia/addons/tasks.py:244]: despite our warnings in settings_base, I guess this is okay @diox ?

**Review by bobsilverberg — APPROVED** [review]
Looks fine to me, but I see there is also a question for @diox.

File: src/olympia/addons/tasks.py (hunk 1 above)
Comment (diox) [reply src/olympia/addons/tasks.py:244]: I guess so. It's worth trying out (though maybe confirm with ops if the queues are also separated on dev)

Comment (eviljeff) [reply src/olympia/addons/tasks.py:244]: ops confirmed the queues are separate on addons-dev too

**Pull request merged**